honeybee_schema/_base.py

```diff
@@ -28,12 +28,12 @@ class IDdBaseModel(NoExtraBaseModel):
             'any spaces or special characters.'
     )
 
-    display_name: str = Field(
+    display_name: Optional[str] = Field(
         default=None,
         description='Display name of the object with no character restrictions.'
     )
 
-    user_data: dict = Field(
+    user_data: Optional[dict] = Field(
         default=None,
```

Comment:
Same comment. You don't need optional here. The default is already None which makes it optional.